xtask/src/main.rs

```diff
@@ -40,6 +42,18 @@ pub enum Command {
 impl Xtask {
     pub fn run(&self) -> Result<()> {
         match &self.command {
+            Command::All(commands::All {
+                compliance,
+                lint,
+                test,
+            }) => {
+                eprintln!("Checking format and clippy...");
+                lint.run()?;
+                eprintln!("Checking licenses...");
+                compliance.run()?;
+                eprintln!("Running tests...");
+                test.run()
```

Comment:
I salute the effort of making the `all.rs` file but this should have go to a function called `run()` exactly like the other commands 😁

Reply:
oh 😅 i could lol, true lemme iterate on that